jenkins-scripts/docker/lib/docker_generate_dockerfile.bash

```diff
@@ -267,7 +267,7 @@ DELIM_DOCKER_WORKAROUND_SIMBODY
 fi
 
 # Install debian dependencies defined on the source code
-SOURCE_DEFINED_DEPS="$(sort -u $(find . -iname 'packages-'$DISTRO'.apt' -o -iname 'packages.apt' | grep -v '/\.git/') | tr '\n' ' ')"
+SOURCE_DEFINED_DEPS="$(sort -u $(find ${SOFTWARE_DIR} -iname 'packages-'$DISTRO'.apt' -o -iname 'packages.apt' | grep -v '/\.git/') | tr '\n' ' ')"
```

Comment:
Since `SOFTWARE_DIR` could be empty or unset, let's add a big of logic to workaround that case.

```suggestion
DEPENDENCIES_PATH_TO_SEARCH=${SOFTWARE_DIR:=.}
SOURCE_DEFINED_DEPS="$(sort -u $(find ${DEPENDENCIES_PATH_TO_SEARCH} -iname 'packages-'$DISTRO'.apt' -o -iname 'packages.apt' | grep -v '/\.git/') | tr '\n' ' ')"
```

The change looks good to me a part of this little corner case.